src/components/molecules/PageHeader.module.css

```diff
@@ -1,8 +1,33 @@
+.homeWrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 20rem;
+  margin-bottom: var(--spacer);
+  background-image: url('../../images/euprogigant-home-eyecatcher-scaled.jpeg');
```

Comment:
question(non-blocking): How do you optimize images (size)?

Reply:
The images were provided by EuProGigant, I haven't touched them any further.

Reply:
We should do it. It looks like nobody every touched some of them :D

Reply:
All right, I'll take care of this 😄